rust/rope/src/test_helpers.rs

```diff
@@ -1,7 +1,24 @@
+// Copyright 2016 Google Inc. All rights reserved.
```

Comment:
Should in general be current year for new files, but not worth blocking the merge over. i'll push on green